__test__/api/comment-vote.test.js

```diff
@@ -7,71 +7,64 @@ import { describeIfEndpoint } from "../helpers/conditionalTests";
 
 describeIfEndpoint(
   "PATCH",
-  "/comments/:commentId/votes",
-  "PATCH '/comments/:commentId/votes'",
+  "/comments/:commentId/votes/upvote",
+  "PATCH '/comments/:commentId/votes/upvote'",
   () => {
     test("Should upvote a comment", async () => {
-      const comment = new CommentModel({
+      const commentObject = {
         content: "this is a comment",
         ownerId: "useremail@email.com",
         origin: "123123",
         applicationId: mongoose.Types.ObjectId(),
-      });
+      };
+      const comment = new CommentModel(commentObject);
       await comment.save();
 
-      const res = await request.patch(`/comments/${comment._id}/votes`).send({
-        voteType: "upvote",
-      });
+      const res = await request
+        .patch(`/comments/${comment._id}/votes/upvote`)
+        .send({
+          ownerId: commentObject.ownerId,
+        });
       if (res.status === 404) {
         return true; // route not implemented yet
       }
       expect(res.status).toBe(200);
-      expect(res.body.data.commentId).toBeTruthy();
-      expect(res.body.data.totalVotes).toBeTruthy();
-      expect(res.body.data.upvotes).toBeTruthy();
-      expect(res.body.data.downvotes).toBeTruthy();
+      expect(res.body.data.commentId).toEqual(String(comment._id));
+      expect(res.body.data.numOfVotes).toEqual(1);
+      expect(res.body.data.numOfUpVotes).toEqual(1);
+      expect(res.body.data.numOfDownVotes).toEqual(0);
     });
+  }
+);
 
+describeIfEndpoint(
+  "PATCH",
+  "/comments/:commentId/votes/downvote",
+  "PATCH '/comments/:commentId/votes/downvote'",
+  () => {
     test("Should downvote a comment", async () => {
-      const comment = new CommentModel({
+      const commentObject = {
         content: "this is a comment",
         ownerId: "useremail2@email.com",
         origin: "123123",
         applicationId: mongoose.Types.ObjectId(),
-      });
+      };
+      const comment = new CommentModel(commentObject);
       await comment.save();
 
-      const res = await request.patch(`/comments/${comment._id}/votes`).send({
-        voteType: "downvote",
-      });
+      const res = await request
+        .patch(`/comments/${comment._id}/votes/downvote`)
+        .send({
+          ownerId: commentObject.ownerId,
+        });
       if (res.status === 404) {
```

Comment:
the utility function `describeIfEndpoint` handles 404 error, so you can completely remove the `If` block